tests/analysis/damages/test_damages.py

```diff
@@ -380,7 +380,9 @@ def test_risk_calculation_default(self, risk_data_file: Path):
         damage_network = DamageNetworkReturnPeriods.construct_from_csv(
             risk_data_file, sep=";", representative_damage_percentage=100
         )
-        damage_network.control_risk_calculation(mode=RiskCalculationModeEnum.DEFAULT)
+        damage_network.control_risk_calculation(
+            mode=RiskCalculationModeEnum.DEFAULT, year=0
```

Comment:
I see year =0 here, are you dividing later on by year?

Reply:
Ow I did a mistake. I started to change 513-resolve_zero_division_in_triangle_to_null by mistake instead of another branch. 513-resolve_zero_division_in_triangle_to_null was supposed to deal with the zero division. 

Sorry for this